spec/page/get_weekly_metrics_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+RSpec.describe 'Fb::Page#get_weekly_metrics' do
+  let(:user) { Fb::User.new access_token: ENV['FB_TEST_ACCESS_TOKEN'] }
+  let(:page) { user.pages.first  }
+
+  context 'given a page and valid metrics' do
+    let(:metrics) { %i(page_impressions page_impressions_paid) }
+
+    it 'returns a hash of given metrics mapped to their values' do
+      weekly_metrics = page.get_weekly_metrics metrics
+      expect(weekly_metrics).to be_a(Hash)
+      expect(weekly_metrics.keys).to all (be_a Symbol)
+      expect(weekly_metrics.keys.sort).to eq metrics.sort
+      expect(weekly_metrics.values).to all (be_an Integer)
+    end
+
+    it 'returns metrics mapped to their values if given an until date' do
+      weekly_metrics = page.get_weekly_metrics metrics, until: Date.today - 7
+      expect(weekly_metrics).to be_a(Hash)
+      expect(weekly_metrics.keys).to all (be_a Symbol)
+      expect(weekly_metrics.keys.sort).to eq metrics.sort
+      expect(weekly_metrics.values).to all (be_an Integer)
+    end
```

Comment:
@S1v4 There is no need to duplicate all the tests for a single option.
You could write a different test just to provide that `until` has some effect.

For instance, I would expect some metric "until 2013" to always return 0 ?